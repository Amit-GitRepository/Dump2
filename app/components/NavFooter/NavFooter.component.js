import bottomNavShadow from '../../assets/bottomNavShadow.png';
import NavFooterItem from './NavFooterItem.component';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './NavFooter.style';
import {Image, Platform, View} from 'react-native';

class NavFooter extends Component {
  DropShadowComponent = Platform.OS === 'android' ? <Image style={styles.androidShadowComponent} source={bottomNavShadow}/> : null;
  render () {
    const {currentScreen, goToScreen} = this.props;
    const footerConfig = [{
      icon: 'bitcoin',
      title: 'SIDE_MENU__BILLS_USAGE',
      screen: 'BillUsage'
    }, {
      icon: 'buy-extra',
      title: 'SIDE_MENU__BUY_EXTRA_PACKAGE',
      screen: 'BuyExtra'
    }, {
      icon: 'true-chat',
      title: 'SIDE_MENU__CHAT',
      screen: 'TrueChat'
    }];
    footerConfig.forEach((footerItem) => {
      footerItem.isActiveScreen = currentScreen === footerItem.screen;
      return footerItem;
    });
    return (
      <View style={styles.container}>
        {this.DropShadowComponent}
        <View style={styles.navRowContainer}>
          {
            footerConfig.map((item, index) => <NavFooterItem key={index} {...item} style={{}} goToScreen={goToScreen} isLastItem={index === footerConfig.length - 1}/>)
          }
        </View>
      </View>
    );
  }
}

NavFooter.propTypes = {
  goToScreen: PropTypes.func,
  currentScreen: PropTypes.string
};

export default NavFooter;
