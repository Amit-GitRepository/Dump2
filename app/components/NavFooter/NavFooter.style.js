import {colors} from '../../themes/constants.styles';

export default {
  container: {
    backgroundColor: colors.TRANSPARENT,
    height: 65,
    position: 'absolute',
    left: 0,
    right: 0,
    bottom: 0,
    width: '100%'
  },
  androidShadowComponent: {
    width: '100%',
    height: 10,
    position: 'absolute',
    top: -3,
    backgroundColor: colors.TRANSPARENT
  },
  navRowContainer: {
    marginTop: 5,
    height: 60,
    flexDirection: 'row',
    backgroundColor: colors.PRIMARY_BG_TEXT_CONTRAST,
    shadowColor: colors.SHADOW,
    shadowOpacity: 0.2,
    shadowRadius: 2,
    shadowOffset: {
      height: -2
    },
    zIndex: 2
  }
};
