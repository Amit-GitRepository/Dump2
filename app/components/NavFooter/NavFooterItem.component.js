import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './NavFooterItem.style';
import {colors, fonts} from '../../themes/constants.styles';
import {Dimensions, View} from 'react-native';
import {Icon, ISText, Touchable} from '../Shared';
import {translate} from '../../language/i18n/helper';

class NavFooterItem extends Component {
  navigateToScreen = (screen) => () => {
    this.props.goToScreen(screen);
  }

  render () {
    const {icon, screen, title, isActiveScreen, isLastItem} = this.props;
    const wrapperStyle = (Dimensions.get('window').width > 350) ? styles.wrapper : styles.wrapperSmallDevice;
    return (
      <Touchable style={isActiveScreen ? [wrapperStyle, styles.activeTab] : wrapperStyle} onPress={this.navigateToScreen(screen)}>
        <View style={isLastItem ? styles.container : [styles.container, styles.separator]}>
          <Icon name={icon} size={fonts.FONT_SIZE_XXL} color={isActiveScreen ? colors.PRIMARY_BG_TEXT_CONTRAST : colors.PRIMARY_TEXT_TAB_LABEL}/>
          <ISText type='BOLD' style={isActiveScreen ? [styles.tabTitle, styles.activeTabTitle] : styles.tabTitle}>{translate(title)}</ISText>
        </View>
      </Touchable>

    );
  }
}

NavFooterItem.propTypes = {
  icon: PropTypes.string,
  title: PropTypes.string,
  screen: PropTypes.string,
  goToScreen: PropTypes.func,
  isActiveScreen: PropTypes.bool,
  isLastItem: PropTypes.bool
};

export default NavFooterItem;
