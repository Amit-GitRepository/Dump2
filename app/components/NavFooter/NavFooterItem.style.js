import {colors, fonts} from '../../themes/constants.styles';

export default {
  wrapper: {
    flex: 1 // Requirement: Item size equal in large device
  },
  wrapperSmallDevice: {
    flexGrow: 1
  },
  container: {
    alignItems: 'center',
    padding: 4
  },
  separator: {
    borderRightWidth: 2,
    borderColor: colors.PRIMARY_BG_SEPARATOR
  },
  activeTab: {
    backgroundColor: colors.PRIMARY_ACTIONABLE
  },
  tabTitle: {
    color: colors.PRIMARY_TEXT_TAB_LABEL,
    fontSize: fonts.FONT_SIZE_SMALL
  },
  activeTabTitle: {
    color: colors.PRIMARY_BG_TEXT_CONTRAST
  }
};
