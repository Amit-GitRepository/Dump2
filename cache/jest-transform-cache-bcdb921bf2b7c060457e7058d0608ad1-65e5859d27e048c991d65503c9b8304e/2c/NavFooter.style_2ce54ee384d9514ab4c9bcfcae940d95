Object.defineProperty(exports,"__esModule",{value:true});var cov_2pgdpy3zka=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/NavFooter/NavFooter.style.js',hash='bd010af6831c1bf23eb8146e1b1b0f3a8d15d5a9',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/NavFooter/NavFooter.style.js',statementMap:{},fnMap:{},branchMap:{},s:{},f:{},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _constants=require('../../themes/constants.styles');exports.default=

{
container:{
flexDirection:'row',
backgroundColor:_constants.colors.PRIMARY_BG_TEXT_CONTRAST,
shadowColor:_constants.colors.SHADOW,
shadowOpacity:0.2,
shadowRadius:2,
shadowOffset:{
height:-2}}};