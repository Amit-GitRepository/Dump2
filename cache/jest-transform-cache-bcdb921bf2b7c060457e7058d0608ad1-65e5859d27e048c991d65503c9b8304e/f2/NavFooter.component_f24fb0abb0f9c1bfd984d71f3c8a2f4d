Object.defineProperty(exports,"__esModule",{value:true});var cov_17lr77epkx=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/NavFooter/NavFooter.component.js',hash='e480d509181ddfe776f638857383dd7da4724d36',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/NavFooter/NavFooter.component.js',statementMap:{'0':{start:{line:9,column:40},end:{line:9,column:50}},'1':{start:{line:10,column:25},end:{line:22,column:6}},'2':{start:{line:23,column:4},end:{line:26,column:7}},'3':{start:{line:24,column:6},end:{line:24,column:70}},'4':{start:{line:25,column:6},end:{line:25,column:24}},'5':{start:{line:27,column:4},end:{line:33,column:6}},'6':{start:{line:30,column:44},end:{line:30,column:153}},'7':{start:{line:37,column:0},end:{line:40,column:2}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:8,column:2},end:{line:8,column:3}},loc:{start:{line:8,column:12},end:{line:34,column:3}},line:8},'1':{name:'(anonymous_1)',decl:{start:{line:23,column:25},end:{line:23,column:26}},loc:{start:{line:23,column:41},end:{line:26,column:5}},line:23},'2':{name:'(anonymous_2)',decl:{start:{line:30,column:27},end:{line:30,column:28}},loc:{start:{line:30,column:44},end:{line:30,column:153}},line:30}},branchMap:{},s:{'0':0,'1':0,'2':0,'3':0,'4':0,'5':0,'6':0,'7':0},f:{'0':0,'1':0,'2':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _extends=Object.assign||function(target){for(var i=1;i<arguments.length;i++){var source=arguments[i];for(var key in source){if(Object.prototype.hasOwnProperty.call(source,key)){target[key]=source[key];}}}return target;};var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _NavFooterItem=require('./NavFooterItem.component');var _NavFooterItem2=_interopRequireDefault(_NavFooterItem);
var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _NavFooter=require('./NavFooter.style');var _NavFooter2=_interopRequireDefault(_NavFooter);
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

NavFooter=function(_Component){_inherits(NavFooter,_Component);function NavFooter(){_classCallCheck(this,NavFooter);return _possibleConstructorReturn(this,(NavFooter.__proto__||Object.getPrototypeOf(NavFooter)).apply(this,arguments));}_createClass(NavFooter,[{key:'render',value:function render()
{cov_17lr77epkx.f[0]++;var _ref=(cov_17lr77epkx.s[0]++,
this.props),currentScreen=_ref.currentScreen,goToScreen=_ref.goToScreen;
var footerConfig=(cov_17lr77epkx.s[1]++,[{
icon:'bills-and-usage',
title:'SIDE_MENU__BILLS_USAGE',
screen:'BillUsage'},
{
icon:'buy-extra',
title:'SIDE_MENU__BUY_EXTRA_PACKAGE',
screen:'BuyExtra'},
{
icon:'true-chat',
title:'SIDE_MENU__CHAT',
screen:'TrueChat'}]);cov_17lr77epkx.s[2]++;

footerConfig.forEach(function(footerItem){cov_17lr77epkx.f[1]++;cov_17lr77epkx.s[3]++;
footerItem.isActiveScreen=currentScreen===footerItem.screen;cov_17lr77epkx.s[4]++;
return footerItem;
});cov_17lr77epkx.s[5]++;
return(
_react2.default.createElement(_reactNative.View,{style:_NavFooter2.default.container,elevation:5},

footerConfig.map(function(item,index){cov_17lr77epkx.f[2]++;cov_17lr77epkx.s[6]++;return _react2.default.createElement(_NavFooterItem2.default,_extends({key:index},item,{goToScreen:goToScreen,isLastItem:index===footerConfig.length-1}));})));



}}]);return NavFooter;}(_react.Component);cov_17lr77epkx.s[7]++;


NavFooter.propTypes={
goToScreen:_propTypes2.default.func,
currentScreen:_propTypes2.default.string};exports.default=


NavFooter;