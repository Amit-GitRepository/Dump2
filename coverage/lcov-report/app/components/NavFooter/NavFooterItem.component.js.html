<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for app/components/NavFooter/NavFooterItem.component.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">app/components/NavFooter</a> NavFooterItem.component.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>5/5</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">62.5% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>5/8</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">7x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">6x</span>
<span class="cline-any cline-yes">6x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">7x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import Icon from '../Shared/Icon/Icon.component';
import ISText from '../Shared/ISText/ISText.component';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './NavFooterItem.style';
import Touchable from '../Shared/Touchable/Touchable.component';
import {colors, fonts} from '../../themes/constants.styles';
import {translate} from '../../language/i18n/helper';
import {View} from 'react-native';
&nbsp;
class NavFooterItem extends Component {
  navigateToScreen = (screen) =&gt; () =&gt; {
    this.props.goToScreen(screen);
  }
&nbsp;
  render () {
    const {icon, screen, title, isActiveScreen, isLastItem} = this.props;
    return (
      &lt;Touchable style={isActiveScreen ? <span class="branch-0 cbranch-no" title="branch not covered" >[styles.wrapper, styles.activeTab] </span>: styles.wrapper} onPress={this.navigateToScreen(screen)}&gt;
        &lt;View style={isLastItem ? styles.container : [styles.container, styles.separator]}&gt;
          &lt;Icon name={icon} size={fonts.FONT_SIZE_XXL} color={isActiveScreen ? <span class="branch-0 cbranch-no" title="branch not covered" >colors.PRIMARY_BG_TEXT_CONTRAST </span>: colors.PRIMARY_TEXT_TAB_LABEL}/&gt;
          &lt;ISText type='BOLD' style={isActiveScreen ? <span class="branch-0 cbranch-no" title="branch not covered" >[styles.tabTitle, styles.activeTabTitle] </span>: styles.tabTitle}&gt;{translate(title)}&lt;/ISText&gt;
        &lt;/View&gt;
      &lt;/Touchable&gt;
&nbsp;
    );
  }
}
&nbsp;
NavFooterItem.propTypes = {
  icon: PropTypes.string,
  title: PropTypes.string,
  screen: PropTypes.string,
  goToScreen: PropTypes.func,
  isActiveScreen: PropTypes.bool,
  isLastItem: PropTypes.bool
};
&nbsp;
export default NavFooterItem;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Oct 31 2017 11:24:42 GMT+0530 (IST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
