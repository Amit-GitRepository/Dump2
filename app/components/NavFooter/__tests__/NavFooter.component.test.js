import NavFooter from '../NavFooter.component';
import React from 'react';
import ReactSixteenAdapter from 'enzyme/build/adapters/ReactSixteenAdapter';
import renderer from 'react-test-renderer';
import {configure, shallow} from 'enzyme';

configure({adapter: new ReactSixteenAdapter()});

describe('NavFooter component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NavFooter/>);
    const component = renderer.create(<NavFooter />).toJSON();
    expect(component).toMatchSnapshot();
    expect(wrapper).toBeDefined();
  });
});
