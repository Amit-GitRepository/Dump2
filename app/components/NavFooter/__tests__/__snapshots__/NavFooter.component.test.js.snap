// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavFooter component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "bottom": 0,
      "height": 65,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flexDirection": "row",
        "height": 60,
        "marginTop": 5,
        "shadowColor": "#303C46",
        "shadowOffset": Object {
          "height": -2,
        },
        "shadowOpacity": 0.2,
        "shadowRadius": 2,
        "zIndex": 2,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "padding": 4,
              },
              Object {
                "borderColor": "#E9E9EA",
                "borderRightWidth": 2,
              },
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#303C46",
                  "fontSize": 30,
                },
                undefined,
                Object {
                  "fontFamily": "iservice",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#303C46",
                "fontFamily": "SukhumvitSet-Bold",
                "fontSize": 14,
              }
            }
          >
            Bills & Usage
          </Text>
        </View>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "padding": 4,
              },
              Object {
                "borderColor": "#E9E9EA",
                "borderRightWidth": 2,
              },
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#303C46",
                  "fontSize": 30,
                },
                undefined,
                Object {
                  "fontFamily": "iservice",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#303C46",
                "fontFamily": "SukhumvitSet-Bold",
                "fontSize": 14,
              }
            }
          >
            Buy Extra Package
          </Text>
        </View>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "padding": 4,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#303C46",
                  "fontSize": 30,
                },
                undefined,
                Object {
                  "fontFamily": "iservice",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#303C46",
                "fontFamily": "SukhumvitSet-Bold",
                "fontSize": 14,
              }
            }
          >
            True Care
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
